src/renderers/WebGLRenderer.js

```diff
@@ -290,6 +294,7 @@ function WebGLRenderer( parameters ) {
 		}
 
 		extensions.get( 'OES_texture_float_linear' );
+		extensions.get( 'EXT_color_buffer_half_float' );
```

Comment:
The [MDN](https://developer.mozilla.org/en-US/docs/Web/API/EXT_color_buffer_half_float) says that `EXT_color_buffer_half_float` is available to WebGL 1 contexts only. Should it be in the `else` statement above?